sparkle/requirements.txt

```diff
@@ -0,0 +1,49 @@
+attrs==19.3.0
```

Comment:
Not essential, but just curious if `EBS` will accept a `requirements.txt` that lists only high-level packages (and not their dependencies) like `boto3, pandas, etc`.